docs/release_guide.md

```diff
@@ -152,6 +156,25 @@ On the docs site's main branch, edit the file [https://github.com/openservicemes
 1. Copy the table and replace the existing table in the file [https://github.com/openservicemesh/osm-docs/blob/main/content/docs/guides/troubleshooting/control_plane_error_codes.md](https://github.com/openservicemesh/osm-docs/blob/main/content/docs/guides/troubleshooting/control_plane_error_codes.md).
 1. If there were updates to the table, make a PR in the OSM docs repository.
 
+### 5. Update Grafana Dashboards
```

Comment:
This should not be a part of the release workflow, instead should be something we actively do. We need to trim the steps involved during the release to keep it as simple as possible. This also includes API docs generation and other routine tasks. Perhaps the simplest way to achieve this is to create an issue and mark it as a release blocker when we know the dashboards need an update prior to the release being cut. 

I suggest moving this to a new doc called `pre_release_checklist.md` which describes the checks needing to be performed prior to starting the release process. 

We certainly should not expect the release manager to be doing all of this.

Reply:
I've created the following issue to address simplifying the release guide and automating the process of versioning and uploading the OSM dashboards to Grafana.com.
- https://github.com/openservicemesh/osm/issues/4516
- https://github.com/openservicemesh/osm/issues/4515